### PR TITLE
DCOS-48532: No text button in Marathon configuration

### PR DESCRIPTION
**Pull request by GeorgiSTodorov**
Add a text for a textless button.

Closes https://jira.mesosphere.com/browse/DCOS-48532

## Testing
1. Go to catalog.
2. Open `marathon`.
3. Click "Review & Run".
4. Open the "Marathon" tab and scroll to the bottom.
5. Verify that the button is not empty.

## Trade-offs
Somewhat of a band-aid solution.

## Dependencies
None.

### Before
Снимка от 2019-06-12 15-59-26

### After
Снимка от 2019-06-12 15-38-21

### CODE REVIEW

File: src/styles/components/framework-configuration/styles.less [hunk 1]
```diff
@@ -29,3 +29,7 @@
     z-index: 0;
   }
 }
+
+.glyphicon-plus:before {
```
Comment (mperrotti) [inline src/styles/components/framework-configuration/styles.less]: This element is a `<i>`, so the browser styles it as italic.

Could you add `font-style: normal` to these styles?

**Comment by mperrotti** [comment]
I see this original component is coming from `react-jsonschema-form`. Is there a way we can replace it with our `AddButton`?

**Comment by GeorgiSTodorov** [comment]
@mperrotti , where do you see the button from `react-jsonschema-form`?

**Comment by mperrotti** [comment]
@GeorgiSTodorov - by inspecting through the React inspector

**Comment by GeorgiSTodorov** [comment]
@mperrotti , I hadn't considered that. Still, I am not sure if that wouldn't be an over-kill for this bug. I initially tried to use our icons from the ui-kit, but I couldn't get the imports in the `less` file to work, so I decided to use the current solution as good enough.

File: src/styles/components/framework-configuration/styles.less (hunk 1 above)
Comment (GeorgiSTodorov) [reply src/styles/components/framework-configuration/styles.less]: The plus is not _italic_ in the screenshot, is it italic for you?

**Comment by mperrotti** [comment]
Yup, I agree - current solution is fine. Just change the `font-style` from italic :)

File: src/styles/components/framework-configuration/styles.less (hunk 1 above)
Comment (mperrotti) [reply src/styles/components/framework-configuration/styles.less]: @GeorgiSTodorov - yes, it was _italic_ for me in Chrome

**Pull request merged**